docs/CHANGES.TXT

```diff
@@ -64,6 +64,7 @@
 - Fix: Colors of DVB subtitles with depth 2 broken due to a missing break.
 - Fix: CEA-708: Caption loss due to CW command (#991).
 - Fix: CEA-708: Update patch for windows priority with functions (#990).
+- Fix: Update UTF8Proc to 2.2.0
```

Comment:
This should go into a new version (since 0.87 was released)
So add something like
0.88 (unreleased)
-------------------

At the top and then your line.